packages/cactus-test-tooling/src/main/typescript/public-api.ts

```diff
@@ -62,6 +62,14 @@ export {
   K_DEFAULT_VAULT_DEV_ROOT_TOKEN,
 } from "./vault-test-server/vault-test-server";
 
+export {
+  ILocalStackOptions,
+  LocalStack,
```

Comment:
@jagpreetsinghsasan Also take care to rename here if you use the VSCode renaming feature